src/openapi/components/schemas/ApplicationFormField.json

```diff
@@ -31,6 +31,10 @@
 			"type": "string",
 			"example": "Your First Name"
 		},
+		"instructions": {
```

Comment:
Let's add a description to this field so it shows up in documentation for API users.